libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -220,4 +224,70 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
             }),
         );
     }
+
+    public getUriIdentifierObjects = async (identifiers: Identifier[]): Promise<IUriIdentifierObject[]> => {
+        const uriIdentifierPairs = await this.authCall((sdk) =>
+            sdk.md.getUrisFromIdentifiers(this.workspace, identifiers),
+        );
+        return uriIdentifierPairs.map(convertUriIdentifierPair);
+    };
+
+    public getAttributeDisplayFormsByUris = async (
+        uris: Uri[],
+    ): Promise<IAttributeDisplayFormMetadataObject[]> => {
+        const wrappedAttributeOrDisplayForms = await this.authCall((sdk) =>
```

Comment:
so this contains Attribute Or DisplayForms, right?

Reply:
fixed